pycqed/measurement/waveform_control/pulsar.py

```diff
@@ -31,6 +31,8 @@
 try:
     from pycqed.instrument_drivers.physical_instruments.ZurichInstruments.\
         UHFQuantumController import UHFQC
+    from pycqed.instrument_drivers.physical_instruments.ZurichInstruments. \
+        dummy_UHFQC import dummy_UHFQC
```

Comment:
remove

Reply:
Sorry for the laconic comment before. 
This is a duplicate import as the dummyUHFQC is imported on line 45-46

Reply:
I will fix